command_line/find_shared_models.py

```diff
@@ -38,14 +46,8 @@ class Script(object):
 
     def __init__(self):
         """Initialise the script."""
-        from dials.util.options import OptionParser
-        import libtbx.load_env
-
         # The script usage
-        usage = (
-            "usage: %s [options] [param.phil] "
-            "models.expt " % libtbx.env.dispatcher_name
-        )
+        usage = "usage: dials.find_shared_models [options] [param.phil] " "models.expt "
```

Comment:
just as comment: iirc this is equivalent to
```suggestion
        usage = "dials.find_shared_models [options] [param.phil] models.expt"
```
because the OptionParser adds the `usage: ` bit automatically